gallery/src/styles/CssVariables/CssVariablesPage.tsx

```diff
@@ -390,8 +390,9 @@ const CssVariablesPage = () => {
             <ColorDocRow colorVar="--nx-color-threat-indicator-legend-header-text">
               The text color used on the <NxCode>NxThreatIndicatorLegend</NxCode> header.
             </ColorDocRow>
-            <ColorDocRow colorVar="--nx-color-threat-indicator-legend-item-text">
-              The text color used on the <NxCode>NxThreatIndicatorLegend</NxCode> items.
+            <ColorDocRow colorVar="--nx-color-threat-level-text">
```

Comment:
I changed this to `--nx-color-threat-level-text` because both NxThreatCounter & NxThreatIndicatorLegend use the same color variables, so it seemed unnecessary to have two different variables for the each component's text.